### ISSUE TITLE
about the pre-training stage loss function.

### ISSUE THREAD

**Issue by vicentowang** [paTRICK-swk/P-STMO#9]
I feel confused.

paper see.
![image](https://user-images.githubusercontent.com/49265512/183366081-4b5ebc95-6412-4b0f-8d3d-9dfc1ba2e757.png)

but code :
 loss = mpjpe_cal(output_2D, torch.cat((input_2D[:, ~mask], input_2D[:, mask]), dim=1))

what i missed.

**Comment by paTRICK-swk** [comment]
The formula is written in the way of maximum likelihood estimation (MLE, the paper), which is equivalent to mean square error (MSE, the code) under the normal error assumption. FYI, please refer to [this link](https://stats.stackexchange.com/questions/143705/maximum-likelihood-method-vs-least-squares-method).
